dotcom-rendering/src/web/experiments/tests/sign-in-gate-copy-test-variants.ts

```diff
@@ -0,0 +1,29 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const signInGateCopyTestJan2023: ABTest = {
+	id: 'SignInGateCopyTestJan2023',
+	start: '2023-01-23',
+	expiry: '2025-12-01',
+	author: 'Lindsey Dew',
+	description: 'Test varying the copy in the call to action for sign in gate',
+	audience: 0.1,
```

Comment:
Make this 2%

Reply:
Updated audience segmentation as per: https://docs.google.com/document/d/1lBufse0NxTZqRNFFRbch1hcFlyv65gEyNyMHtoqKj48/edit